src/assets/thirdPartyServices/common/MethodNamesDropDown.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const MethodNamesDropDown = ({ list, value, onChange, ...restProps }) => {
+  return (
+    <select
+      name="methodName"
+      value={value}
+      style={{ height: "30px", width: "250px", fontSize: "13px", marginBottom: "5px" }}
+      onChange={onChange}
+      {...restProps}
+    >
+      {list.map(item => (
```

Comment:
Do you think we could get the method names within this component itself rather than passing in the list? The consumer of this component could pass in the Stub itself.

Reply:
We could.. but I'm thinking to make this a common-reusable- dropdown component in the future as a part of custom ui..So it would be agnostic of the list provided to it.

Reply:
Makes sense.